src/CadViewerContainer.tsx

```diff
@@ -55,7 +56,7 @@ export const CadViewerContainer = forwardRef<
         camera={{ up: [0, 0, 1], position: [5, 5, 5] }}
       >
         <RotationTracker />
-        <OrbitControls autoRotate autoRotateSpeed={1} />
+        {!autoRotateDisabled && <OrbitControls autoRotate autoRotateSpeed={1} />}
```

Comment:
this disables all orbital controls, you probably want to change the `autoRotate` prop